app/controllers/concerns/authentication.rb

```diff
@@ -97,42 +152,38 @@ def query_token_authenticator(query_token)
   end
 
   def http_password_authenticator(username = nil, password = nil)
+    if current_account.sso? && current_account.sso_organization_domains.any? { username.ends_with?(_1) } # for JIT-user-provisioning
```

Comment:
Is this the best way to check if the user's email is for one of the org's domains?

Reply:
For example, if the org domain was `actor.example`, `badactor.example` would also match.